server/entity/user.entity.ts

```diff
@@ -0,0 +1,51 @@
+/* tslint:disable:variable-name */
+
+import { Column, Entity, OneToOne, PrimaryGeneratedColumn } from "typeorm";
+import { TimestampedEntity } from "./timestamped.entity";
+import { UserDetails } from "./user-details.entity";
+
+/**
+ * User account information.
+ */
+@Entity()
+export class User extends TimestampedEntity {
+
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @Column({ unique: true })
+  public name: string;
+
+  @Column({ nullable: true })
+  public title: string;
+
+  @Column({ unique: true })
+  public email: string;
+
+  @Column({ nullable: true })
+  public avatar: string;
+
+  @Column({ default: false })
+  public is_mod: string; // XXX "true" or ""
+
+  @Column({ default: false })
+  public is_admin: string; // XXX "true" or ""
```

Comment:
These *really* should be migrated into bools :)

Reply:
I reviewed it during my own CR and it's actually even worse than that! https://github.com/alakajam-team/alakajam/blob/master/server/entity/user.entity.ts#L33